File: Test/author_solution/cpp/Seller.cpp
#include "Seller.hpp"

Seller::Seller(int quantity)
{
    this->quantity=quantity;
}

int Seller::getQuantity()
{
    return quantity;
}

void Seller::setQuantity(int quantity)
{
    this->quantity=quantity;
}

int Seller::sell()
{
    if(quantity>0)
    {
      quantity--;
      return 1;
    }
    else
    {
      return 0;
    }
}

File: docs/examples/unit_tests/author_solution/python2/Seller.py
class Seller:
	quantity = 0

	def __init__(self, quantity=0):
		self.quantity = quantity

	def getQuantity(self):
		return self.quantity

	def setQuantity(self,quantity=0):
		self.quantity = quantity

	def sell(self):
		if self.quantity > 0:
			self.quantity -= 1
			return 1
		else:
			return 0		
